CHANGELOG.md

```diff
@@ -53,7 +58,7 @@ ___Note:__ Yet to be released changes appear here._
 
 ## 1.0.0-alpha.6

```

Comment:
Shall we keep the alpha versions or simply focus on the stable release here?

I'm asking because we recently did the same in bpmn-js: https://github.com/bpmn-io/bpmn-js/commit/d17e175c754cfa4baf4f1b69777a6465a767c49d

Reply:
I could join the alphas but then it's a huge changelog update.

Reply:
Proposal: Summarize most notable changes under the umbrella of _COmplete Rework_.

Reply:
Also, link / show _all_ breaking changes.

Reply:
Already discussed this with @barmac and we will handle it similarly as you proposed 👍